cluster/charts/rook-ceph/values.yaml

```diff
@@ -3,7 +3,10 @@
 # Declare variables to be passed into your templates.
 
 image:
-  prefix: rook
+  registry:
```

Comment:
How about naming these registries according to how rook is naming them? For example:
```yaml
registry:
  rook: docker.io
  sigstorage: k8s.gcr.io
  cephcsi: quay.io
```